docs/man/antler-proj.1

```diff
@@ -0,0 +1,165 @@
+.\" Automatically generated by Pandoc 2.5
+.\"
+.TH "aproj" "1" "March 09, 2023" "aproj 1.0.0" "User Manual"
+.hy
+.SH NAME
+.PP
+aproj \- Antler Project Management System
+.SH SYNOPSIS
+.PP
+\f[C]aproj <command> [<args>]\f[R]
+.SH DESCRIPTION
+.PP
+\f[B]Antler Project Management System\f[R] is a set of libraries and
+tools to help users successfully develop smart contracts.
+.SH COMMANDS
+.PP
+This is a general help text which prints when aproj runs without a
+command or with key \[en]help.
+.PP
+\f[B]Example: aproj \[en]help\f[R]
```

Comment:
antler-proj

Reply:
Change the aproj to antler-proj in all locations.

Reply:
To be addressed in the source file.